acceptance/bundle/artifacts/whl_prebuilt_outside_dynamic/test.toml

```diff
@@ -0,0 +1,5 @@
+Badness = "Expect this to work, but deploy fails: ... is not contained in sync root path"
```

Comment:
Is this still the case with the fix in place?

Reply:
No, good catch. Cleaned up.